src/IEPCreated/IEPCreated.js

```diff
@@ -24,7 +24,7 @@ const IEPCreated = ({ showModal, offender, iepValues, activityName, user }) => {
       offenderNo: offender.offenderNo,
       offenderName: `${properCaseName(offender.firstName)} ${properCaseName(offender.lastName)}`,
       caseNote: `${pascalToString(iepValues.absentReason)}: ${iepValues.comments}`,
-      cellLocation: offender.cellLocation,
+      cellLocation: stripAgencyPrefix(offender.cellLocation, user.activeCaseLoadId),
```

Comment:
not sure this looks correct - why use the `user` active case load rather than the `offender.agencyId`?